src/components/frame/v5/pages/StreamingPaymentsPage/partials/StreamingPaymentsTable/StreamingPaymentsTable.tsx

```diff
@@ -0,0 +1,113 @@
+import { BigNumber } from 'ethers';
+import React from 'react';
+import { defineMessages } from 'react-intl';
+
+import { DEFAULT_TOKEN_DECIMALS } from '~constants';
+import { useGetStreamingPaymentsQuery } from '~gql';
+import { formatText } from '~utils/intl.ts';
+import { getAmountPerValue } from '~utils/streamingPayments.ts';
+import Table from '~v5/common/Table/Table.tsx';
+
+import { useRenderSubComponent, useStreamingTableColumns } from './hooks.tsx';
+import { type StreamingTableFieldModel } from './types.ts';
+
+const displayName =
+  'pages.StreamingPaymentsPage.partials.StreamingPaymentsTable';
+
+const MSG = defineMessages({
+  title: {
+    id: `${displayName}.title`,
+    defaultMessage: 'Streaming Payments',
+  },
+});
+
+const StreamingPaymentsTable = () => {
+  const { data, loading } = useGetStreamingPaymentsQuery();
+  const streamingPayments: StreamingTableFieldModel[] = (
+    data?.searchStreamingPayments?.items || []
+  ).reduce<StreamingTableFieldModel[]>((result, item) => {
+    if (!item) {
+      return result;
+    }
```

Comment:
There is a lot happening here, could this be split into multiple steps? You can use `filter(notNull)` to remove the need for `if(!item)` check. It would also be good to indicate the array no longer contains just streaming payments, but a grouping of them by recipient